components/search/SearchSelectedFacets.vue

```diff
@@ -7,7 +7,7 @@
       class="mr-2"
       data-qa="filter badge"
     >
-      {{ selectedFacet.facetName | searchFacetHeader }}: {{ selectedFacet.fieldValue }}
+      {{ $t(`facets.${selectedFacet.facetName}`) }}: {{ selectedFacet.fieldValue }}
```

Comment:
Use [named formatting](http://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) with the colon and a placeholder for the facet field value included.

Reply:
that would make the translation unusable in the facet section headers, so would require unnecessary duplication of the same translation.